src/main/webapp/js/item_page_loader.js

```diff
@@ -0,0 +1,41 @@
+let map;
+function createMap(user_lat, user_lng, user_title) {
+  // create map object and place in map element
+  map = new google.maps.Map(document.getElementById("map"), {

```

Comment:
I'm guessing you're using a script tag for importing google maps, so ignore no-undef issues

Reply:
correct